Makefile

```diff
@@ -805,6 +829,40 @@ deb: install themispp_install themis_jni_install
 
 	@find $(BIN_PATH) -name \*.deb
 
+# TODO: Add Themis as dependency?
+# TODO: Put inside `deb` target?
+# Use builtin feature of fpm to create a .deb package from a Python package dir.
+# Dependencies are automatically added, i.e. PyThemis depends on `six`, so fpm will add `python3-six` to deps.
+pythemis_deb: DEB_ARCHITECTURE = all
```

Comment:
we already have targets `deb` and `deb_php`. Let's keep naming conventions and name it `deb_python`